tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_local_backend/main.tf

```diff
@@ -179,4 +185,141 @@ resource "null_resource" "sam_metadata_aws_lambda_layer_version_from_local" {
         null_resource.build_layer_version
     ]
 }
-## */
\ No newline at end of file
+## */
+
+## /* hello world code builder
+resource "null_resource" "build_hello_world_lambda_function" {
+    triggers = {
+        build_number = "${timestamp()}"
+    }
+
+    provisioner "local-exec" {
+        command = "./py_build.sh \"${local.hello_world_function_src_path}\" \"${local.building_path}\" \"${local.hello_world_artifact_file_name}\" Function"
```

Comment:
Would be nice to have integration tests that use Lambda Builders in the future too